common/defined_error.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef ERROR_H
```

Comment:
Change `ERROR_H` to `COMMON_DEFINED_ERROR_H`.

Reply:
The defined_error.h will be integrated into ta_error.h as soon as possible.

Reply:
Never say "soon" too early. We need clean codebase.